linalg/sparsemat.cpp

```diff
@@ -666,6 +666,7 @@ const real_t &SparseMatrix::operator()(int i, int j) const
 
    if (Finalized())
    {
+      HostReadI(), HostReadJ(), HostReadData();
```

Comment:
Ditto

Reply:
done in https://github.com/mfem/mfem/pull/4675/commits/ea041467ae1ad6d007ea2a6e7880fbfa448f50a7